src/dbnode/server/server.go

```diff
@@ -411,13 +411,16 @@ func Run(runOpts RunOptions) {
 
 	// Setup query stats tracking.
 	statsOpts := stats.QueryStatsOptions{
-		Lookback: stats.DefaultLookback,
+		MaxDocsLookback:      stats.DefaultLookback,
+		MaxBytesReadLookback: stats.DefaultLookback,
 	}
 	if max := runOpts.Config.Limits.MaxRecentlyQueriedSeriesBlocks; max != nil {
-		statsOpts = stats.QueryStatsOptions{
-			MaxDocs:  max.Value,
-			Lookback: max.Lookback,
-		}
+		statsOpts.MaxDocs = max.Value
+		statsOpts.MaxDocsLookback = max.Lookback
+	}
+	if max := runOpts.Config.Limits.MaxRecentlyQueriedSeriesBytesRead; max != nil {
+		statsOpts.MaxBytesRead = max.Value
+		statsOpts.MaxBytesReadLookback = max.Lookback
```

Comment:
These should be validated somewhere; I think cleanest way is to add a `Validate()` method on `QueryStatsOptions` that ensures Values are >=0 and Lookbacks are >0